src/colornames.csv

```diff
@@ -15353,6 +15395,7 @@ Lone Star,#c09458,
 Lonely Chocolate,#4a0a00,x
 Lonely Road,#947754,
 Lonestar,#522426,x
+Lonetree,#7c5b4d,

```

Comment:
Lets remove that one can't find anything for "Lonetree" expect a place called "Lone Tree" but again its not clear why it would be that color